src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -136,12 +136,20 @@ private void changeCheckInServicePointId(UUID servicePointId) {
     write(representation, "checkinServicePointId", servicePointId);
   }
 
+  private void changeItemStatusToDeclaredLost() {
+    Item item = getItem();
+    if (item != null) {
+      item.changeStatus(ItemStatus.DECLARED_LOST);
+    }
+    changeItemStatus(ItemStatus.DECLARED_LOST.getValue());
```

Comment:
Does this need doing here, or does the repository handle this mapping later?

Reply:
I think it needs be here for now as It appeared that the work of getting rid of json representation from the Loan class would take quite some time. I think it should be done separately.